olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/VersionedLayerClient.h

```diff
@@ -41,12 +42,10 @@ class DATASERVICE_READ_API VersionedLayerClient final {
  public:
   /// DataResult alias
   using DataResult = model::Data;
-  /// ApiResponse template alias
-  template <typename Result>
-  using CallbackResponse = client::ApiResponse<Result, client::ApiError>;
-  /// Callback template alias
-  template <typename Response>
-  using Callback = std::function<void(CallbackResponse<Response> response)>;
+  /// CallbackResponse alias
+  using CallbackResponse = client::ApiResponse<DataResult, client::ApiError>;
+  /// Callback alias
+  using Callback = std::function<void(CallbackResponse response)>;
```

Comment:
This is not possible.
How will be the others called for GetPartition() and GetVersion(), etc.?
Why did we change from template?

Reply:
The template was meaningless since there is no other specifications apart from <DataResult>
Once we will need the template, it will be moved somewhere on common space, but this is a subject for another task.